features/signal/basic/README.md

```diff
@@ -0,0 +1,7 @@
+# Signal a workflow
```

Comment:
Ignore all changes from here onwards; they will disappear when we merge https://github.com/temporalio/features/pull/349

Reply:
:+1: Will wait for that merge before approving this one

Reply:
This one's mergeable now.